test/data/setup_data.sh

```diff
@@ -79,3 +79,8 @@ done
 gdal_translate -of GTiff -co NBITS=16 -ot Float32 initial.tiff float_n_bit_16.tiff || true
 gdal_translate -of GTiff -co NBITS=16 -ot Float32 -co TILED=YES initial.tiff float_n_bit_tiled_16.tiff || true
 gdal_translate -of GTiff -co NBITS=16 -ot Float32 -co INTERLEAVE=BAND initial.tiff float_n_bit_interleave_16.tiff || true
+
+# GDAL_METADATA support
+wget https://github.com/GeoTIFF/test-data/archive/refs/heads/main.zip -O geotiff-test-data.zip
```

Comment:
I think we should reference a tag or a commit hash here. e.g: https://github.com/GeoTIFF/test-data/archive/8ac198032d8b02160049ca161e8108e3d38176f3.zip

Reply:
Sorry. I missed this.  I'll do that now.

Reply:
Should be fixed now with https://github.com/geotiffjs/geotiff.js/pull/245/commits/98921d30a4e13e45f99e48b52d789d7be71a6785